server/monitor.go

```diff
@@ -98,30 +101,31 @@ const (
 
 // ConnInfo has detailed information on a per connection basis.
 type ConnInfo struct {
-	Cid            uint64     `json:"cid"`
-	IP             string     `json:"ip"`
-	Port           int        `json:"port"`
-	Start          time.Time  `json:"start"`
-	LastActivity   time.Time  `json:"last_activity"`
-	Stop           *time.Time `json:"stop,omitempty"`
-	Reason         string     `json:"reason,omitempty"`
-	RTT            string     `json:"rtt,omitempty"`
-	Uptime         string     `json:"uptime"`
-	Idle           string     `json:"idle"`
-	Pending        int        `json:"pending_bytes"`
-	InMsgs         int64      `json:"in_msgs"`
-	OutMsgs        int64      `json:"out_msgs"`
-	InBytes        int64      `json:"in_bytes"`
-	OutBytes       int64      `json:"out_bytes"`
-	NumSubs        uint32     `json:"subscriptions"`
-	Name           string     `json:"name,omitempty"`
-	Lang           string     `json:"lang,omitempty"`
-	Version        string     `json:"version,omitempty"`
-	TLSVersion     string     `json:"tls_version,omitempty"`
-	TLSCipher      string     `json:"tls_cipher_suite,omitempty"`
-	AuthorizedUser string     `json:"authorized_user,omitempty"`
-	Account        string     `json:"account,omitempty"`
-	Subs           []string   `json:"subscriptions_list,omitempty"`
+	Cid            uint64      `json:"cid"`
+	IP             string      `json:"ip"`
+	Port           int         `json:"port"`
+	Start          time.Time   `json:"start"`
+	LastActivity   time.Time   `json:"last_activity"`
+	Stop           *time.Time  `json:"stop,omitempty"`
+	Reason         string      `json:"reason,omitempty"`
+	RTT            string      `json:"rtt,omitempty"`
+	Uptime         string      `json:"uptime"`
+	Idle           string      `json:"idle"`
+	Pending        int         `json:"pending_bytes"`
+	InMsgs         int64       `json:"in_msgs"`
+	OutMsgs        int64       `json:"out_msgs"`
+	InBytes        int64       `json:"in_bytes"`
+	OutBytes       int64       `json:"out_bytes"`
+	NumSubs        uint32      `json:"subscriptions"`
+	Name           string      `json:"name,omitempty"`
+	Lang           string      `json:"lang,omitempty"`
+	Version        string      `json:"version,omitempty"`
+	TLSVersion     string      `json:"tls_version,omitempty"`
+	TLSCipher      string      `json:"tls_cipher_suite,omitempty"`
+	AuthorizedUser string      `json:"authorized_user,omitempty"`
+	Account        string      `json:"account,omitempty"`
+	Subs           []string    `json:"subscriptions_list,omitempty"`
+	SubsDetail     []SubDetail `json:"subscriptions_list_detail,omitempty"`
```

Comment:
I would have made it a `[]*SubDetail` in case we ever need to sort those (since []SubDetail would force move of SubDetail struct inside the array), but I see that we also use []SubDetail in subz, so...